jenkins/ci.suse.de/pipelines/openstack-ardana-gerrit.Jenkinsfile

```diff
@@ -16,21 +16,53 @@ pipeline {
 
   stages {
 
-    stage('validate commit message') {
-      steps {
-        script {
-          currentBuild.displayName = "#${BUILD_NUMBER}: ${gerrit_change_ids}"
+    stage('static analysis') {
+      parallel {
+        stage('validate commit message') {
+          steps {
+            script {
```

Comment:
removing the workspace - the `cleanWs` bit - and setting the current build name for that matter, shouldn't be part of a parallel block, because these are global tasks that need to be executed at the very beginning of the pipeline, because they affect all following stages. You should probably create a separate stage for those to be executed before anything else.